schema/to30.xsl

```diff
@@ -37,71 +37,81 @@
 	doctype-public="-//Recordare//DTD MusicXML 3.0 Partwise//EN" />
 
   <!--
-    For the root, only look for score-partwise. Anything else 
+    For the root, only look for score-partwise. Anything else
     as a root element gets ignored.
-  -->  
+  -->
   <xsl:template match="/">
     <xsl:apply-templates select="./score-partwise"/>
   </xsl:template>
 
   <!--
-    Transformations that remove post-3.0 elements and 
+    Transformations that remove post-3.0 elements and
     attributes.
   -->
-  
+
   <!-- Additions in note.mod -->
 
-  <!-- 
-    Remove accidental and accidental-mark elements with 
+  <!--
+    Remove accidental and accidental-mark elements with
     the new other value.
   -->
-  <xsl:template 
+  <xsl:template
     match="accidental[.='other']"/>
- 
-  <xsl:template 
+
+  <xsl:template
     match="accidental-mark[.='other']"/>
-  
-  <xsl:template 
+
+  <xsl:template
     match="accidental/@smufl | accidental-mark/@smufl"/>
-  
+
   <!-- Additions in attributes.mod -->
 
   <!-- Remove key-accidental elements with new other value -->
 
-  <xsl:template 
+  <xsl:template
     match="key-accidental[.='other']"/>
-  
-  <xsl:template 
+
+  <xsl:template
     match="key-accidental/@smufl"/>
-  
+
   <!-- Additions in barline.mod -->
 
   <!-- Additions in common.mod -->
 
+	<!-- Remove n, pf, and sfzp elements -->
+	<xsl:template
```

Comment:
These could all go in one xsl:template with a match of ="n | pf | sfzp". Could the indentation also please match the others? (Maybe this is a GitHub display thing, but in case it isn't...)